bin/docker/Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM java:openjdk-7-jre
+FROM java:openjdk-7-jre-alpine
 
-ENV LC_ALL C
-ENV LANG C.UTF-8
-ENV DEBIAN_FRONTEND noninteractive
-ENV DEBCONF_NONINTERACTIVE_SEEN true
+ENV JAVA_HOME=/usr/lib/jvm/default-jvm
+ENV PATH /usr/local/bin:$PATH
+
+RUN apk add --update bash openjdk7-jre java-cacerts && \
+    ln -sf "${JAVA_HOME}/bin/"* "/usr/bin/"
+
+# fix broken cacerts
+RUN rm -f /usr/lib/jvm/default-jvm/jre/lib/security/cacerts && \
+    ln -s /etc/ssl/certs/java/cacerts /usr/lib/jvm/default-jvm/jre/lib/security/cacerts
```

Comment:
you don't need this extra stuff in here such as the ENV settings, the cacerts stuff, and since we are using the openjdk image java is already there :wink: 

when i tested this out the only line I added from the original file was `RUN apk add --update bash` which should be all we need.